src/EditorFeatures/Core/Diagnostics/AbstractPushOrPullDiagnosticsTaggerProvider.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.CodeAnalysis.Diagnostics;
 internal static class DiagnosticTaggingOptions
```

Comment:
>DiagnosticTaggingOptions [](http://example.com/codeflow?start=22&length=24)

Rename to `DiagnosticTaggingOptionsStorage`

Reply:
Renamed most of the *.Options  to *.OptionsStorage. (Except a few in the CodeStyleLayer, has too many references)